### PR TITLE
docs: add a new Angular section

### PR DESCRIPTION
**Pull request by manekinekko**
Hi, I just added a new Angular section where folks can include more newsletter related to the Angular ecosystem.

Cheers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,9 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 - [JSter](http://jster.net/blog). No nonsense list of curated JavaScript links to your mailbox once every two weeks. [Subscribe](https://us7.list-manage.com/subscribe?u=ed40c0084a0c5ba31b3365d65&id=ec6f32bf5e).
 - [Best of JavaScript](https://weekly.bestofjs.org/) Weekly rankings about the most popular open-source projects related to Node.js and the web platform.
 
+#### Angular
+- [Angular Newsletter](http://cur.ng-newsletter.com/) The free, weekly newsletter of the best Angular content on the web. Hand-picked content by the experts in Angular - delivered directly to your inbox.
```
Comment (ferhatelmas) [inline README.md:104]: returns 404 for me

**Comment by ferhatelmas** [comment]
it's dead, closing.

**Comment by manekinekko** [comment]
Yes, their website seems to be discontinued. Sorry about that.

**Comment by ferhatelmas** [comment]
No worries. Thanks for helping :+1: 

If you want to add some others, feel free to check [issues](https://github.com/zudochkin/awesome-newsletters/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc), there were some suggestions.

**Pull request closed without merging**